platform/admin/core/admin-core-logviewer/src/test/js/filter.js

```diff
@@ -63,5 +81,5 @@ test('compound filter (logical AND)', function (t) {
 
   const filtered = filter({ level: 'DEBUG', message: 'second' }, logs)
 
-  t.equal(filtered.length, 0)
+  t.equal(filtered.length, 1)
```

Comment:
Ya, I decided to change all the tests in this file to check for the specific logs instead of the length, pushing up the changes now